src/services/mongoose/schema.types.d.ts

```diff
@@ -56,8 +56,13 @@ type OnChainMongooseType = {
   address: string
 }
 
+interface DbMetadataRecord {
+  _id: ObjectId
+  routingFeeLastEntry: Date
+}
+
 interface WalletInvoiceRecord {
-  id: string
+  _id: string
```

Comment:
Should this be an `ObjectId`?

Reply:
no, it is a hash (custom id type) so must be a string